src/pythae/models/vq_vae/vq_vae_utils.py

```diff
@@ -50,14 +50,14 @@ def forward(self, z: torch.Tensor, uses_ddp: bool = False):
         commitment_loss = F.mse_loss(
             quantized.detach().reshape(-1, self.embedding_dim),
             z.reshape(-1, self.embedding_dim),
-            reduction="mean",
+            reduction="sum",
         )
 
         embedding_loss = F.mse_loss(
             quantized.reshape(-1, self.embedding_dim),
             z.detach().reshape(-1, self.embedding_dim),
-            reduction="mean",
-        ).mean(dim=-1)
```

Comment:
@clementchadebec I'm unsure if second mean needs removing, I believe it does?

Reply:
@ctr26, yes I think it should be removed since the losses are averaged across the batch in `loss_function` method
https://github.com/clementchadebec/benchmark_VAE/blob/18fe279a16e9ba7db81197cb94c1e91d7d795052/src/pythae/models/vq_vae/vq_vae_model.py#L122-L133